monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -70,7 +69,11 @@ class ConfigurePageComponent extends AuthComponent {
       cnc: {},
       network: {},
       exploits: {},
-      internal: {}
+      internal: {
+        general: {
+          started_on_island: {'ui:widget': 'hidden'}
```

Comment:
Would be nice to add this dynamically, but couldn't find an easy way

Reply:
Will wait for Configuration page rework.